analyzers/src/SonarAnalyzer.CSharp/Rules/ReturnEmptyCollectionInsteadOfNull.cs

```diff
@@ -107,6 +105,8 @@ private static ArrowExpressionClauseSyntax GetExpressionBody(SyntaxNode node)
                 case SyntaxKindEx.LocalFunctionStatement:
                     return ((LocalFunctionStatementSyntaxWrapper)node).ExpressionBody;
 
+                case SyntaxKind.OperatorDeclaration:
+                    return ((OperatorDeclarationSyntax)node).ExpressionBody;
                 default:
```

Comment:
Maybe we should consider adding an empty line between the return statement and the default

Reply:
i rewrote these two functions (GetBody, GetExpressionBody) in a more succinct and readable way, so please review them again :)